end-to-end-tests/build.gradle

```diff
@@ -46,6 +46,7 @@ task testEndToEnd(type: Test) {
     dependsOn ":scheduling-manager:deploy"
 
     doFirst {
+        systemProperty 'blox.tests.region', stack.region.toString()
         systemProperty 'blox.tests.awsProfile', stack.profile.toString()
```

Comment:
For the integ tests, I actually started using the AWS SDK override properties instead of plumbing these through manually: 

https://github.com/blox/blox/blob/d4017f64a0be92a1e83db34956c9926c8d7ded43/integ-tests/build.gradle#L12-L21

That should let us just create default clients everywhere, which should simplify `BloxStack` somewhat. Do you think that pattern is worth adopting here?

Reply:
That looks good. Let me adopt this approach.